.github/workflows/release.yml

```diff
@@ -6,13 +6,13 @@ jobs:
   publish-egg:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: denolib/setup-deno@master
-      - run: deno install -A -f --unstable -n eggs https://x.nest.land/eggs@0.2.2/mod.ts
-      - run: |
+     - uses: actions/checkout@v2
+     - uses: denolib/setup-deno@master
+     - run: deno install -A -f --unstable -n eggs https://x.nest.land/eggs@0.3.2/eggs.ts
+     - run: |
```

Comment:
Only one space indent, this would fail right? looks like how it was previously was correct

(note im only talking about the indents)

Reply:
fix typo b591e0d
fix indentation 26a30bc